manifests/globals.pp

```diff
@@ -111,10 +111,10 @@
     'Suse' => $::operatingsystem ? {
       'SLES' => $::operatingsystemrelease ? {
         /11\.[0-4]/ => '91',
-        default => '93',
+        default => '94',
```

Comment:
@mmoll turns out we need a separate condition. SLES 12.0 needs to stay at 93 but SLES =>12.2 works with 94.